docs/DeveloperGuide.adoc

```diff
@@ -906,8 +906,12 @@ Use case resumes at step 6.
 .  Should work on any link:#mainstream-os[mainstream OS] as long as it has Java `1.8.0_60` or higher installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-{More to be added}
+.  Technical requirements: The Addressbook should be able to work on both 32 and 64 bit system
+.  Performance requirements: The Addressbook should be able to give a response within 1 second
+.  Security requirements: The Addressbook should be able to hide personal
+   information using a password
+.  Quality requirements: The Addressbook should be simple enough for a novice who does not use an addressbook before to use
+.  Privacy requirements: The Addressbook data should only be read and modified by authorized party
```

Comment:
The constraint that data files remain human-readable means that this NFR can't be fufilled. (Can't stop people from editing files they can read!)

Reply:
What I meant was due to the password system, only the ones that is the owner of that account is able to modify the addressbook. Essentially implying that the security system should be strong enough to prevent unintentional privacy leaks.

Reply:
I don't believe that strong (or any, really) security is possible with human-readable data files :(